lib/fizz_buzz.rb

```diff
@@ -0,0 +1,17 @@
+class FizzBuzz
+
+  def self.divisible_by(number)
+    if number == 0
+      0
```

Comment:
This first condition is no longer needed, as it is satisfied by the `else` 🌞

Reply:
I added it in because 0 divided by 15 is zero! So when zero was tested, it came back with Fizz Buzz instead of zero. Is there another way to solve this problem?